src/autogluon/bench/eval/evaluation/evaluate_results.py

```diff
@@ -194,8 +199,12 @@ def evaluate(
     )
 
     if output_dir:
-        save_pd.save(path=output_dir + "results_ranked_all.csv", df=results_ranked_all)
-        save_pd.save(path=output_dir + "results_ranked_by_dataset_all.csv", df=results_ranked_by_dataset_all)
+        save_pd.save(path=os.path.join(output_dir, "results_ranked_all.csv"), df=results_ranked_all)
+        logger.info(f'{os.path.join(output_dir, "results_ranked_all.csv")} saved.')
```

Comment:
If you use a value twice, make it a variable. This prevents bugs / inconsistencies.